openbb_platform/extensions/commodity/integration/test_commodity_api.py

```diff
@@ -20,38 +20,66 @@ def headers():
     return {"Authorization": f"Basic {base64_bytes.decode('ascii')}"}
 
 
+# @pytest.mark.parametrize(
+#    "params",
+#    [
+#        (
+#            {
+#                "asset": "gold",
+#                "start_date": None,
+#                "end_date": None,
+#                "collapse": None,
+#                "transform": None,
+#                "provider": "nasdaq",
+#            }
+#        ),
+#        (
+#            {
+#                "asset": "silver",
+#                "start_date": "1990-01-01",
+#                "end_date": "2023-01-01",
+#                "collapse": "monthly",
+#                "transform": "diff",
+#                "provider": "nasdaq",
+#            }
+#        ),
+#    ],
+# )
+# @pytest.mark.skip(reason="Resource no longer available. Pending replacement/removal.")
+# def test_commodity_lbma_fixing(params, headers):
```

Comment:
What is the best way to deal with this? Do you want us to return something to the user saying that it is dead for the time being?

Reply:
Nope, the command is now returning a 410 status code.